examples/failing/QualifiedQualifiedImports.purs

```diff
@@ -0,0 +1,9 @@
+-- @shouldFailWith ErrorParsingModule
+module Main where
+
+import Prelude
+
+-- qualified import with qualified imported names
+import qualified Control.Monad.Eff.Console (log) as Console
```

Comment:
I don't think this should fail - this style of import allows you to selectively re-export members from another module that you also want to import as qualified.

So re-exporting `Console` here would only re-export `Console.log`.